src/plugins.ts

```diff
@@ -40,6 +40,7 @@ export interface PluginCreateInfo {
  * The portion of tsserver's Project API exposed to plugins
  */
 export interface Project {
+    getCurrentDirectory: () => string
```

Comment:
Style nit, could this just be a method rather than an arrow function?

getCurrentDirectory(): string

Reply:
Yes you right, I update this

Reply:
:heavy_check_mark: